app/views/fines/index.html.erb

```diff
@@ -1,62 +1,72 @@
 <h1 class="sr-only">Fines</h1>
 
 <% content_for(:navigation) do %>
-  <%= render 'shared/navigation' %>
-  <%= render partial: 'shared/group_tabs', locals: { self_value: number_to_currency(patron.fines.sum(&:owed)), group_value: number_to_currency(patron.group.fines.sum(&:owed)) } if patron_or_group.group? %>
+<%= render 'shared/navigation' %>
```

Comment:
Can you restore the indent here?  It makes it easier to see that this is a block body.